ui/package.json

```diff
@@ -43,155 +43,155 @@
     "vault:cluster": "VAULT_REDIRECT_ADDR=http://127.0.0.1:8202 vault server -log-level=error -dev -dev-root-token-id=root -dev-listen-address=127.0.0.1:8202 -dev-ha -dev-transactional"
```

Comment:
There is a lot of bumping done here. My process:
1. Check the current yarn.lock to see what the dependency was being resolved to and bump to that. Confirmed often with npm to make sure that package existed and didn't have special notes.